### ISSUE TITLE
Experiment Settings 

### ISSUE THREAD

**Issue by thsno02** [thsno02/StarGANv2-VC#4]
>We train our model for 150 epochs, with a batch size of 10 two- second long audio segments. We use AdamW [23] optimizer with a learning rate of 0.0001 fixed throughout the training pro- cess. The source classifier joins the training process after 50 epochs. We set $\lambda_{cls}$ = 0.1, $\lambda_{advcls}$ = 0.5, $\lambda_{sty}$ = 1, $\lambda_{advcls}$ = 1, $\lambda_{f0}$ = 5, $\lambda_{asr}$ = 1, $\lambda_{norm}$ = 1 and $\lambda_{cyc}$ = 1. The F0 model is trained with pitch contours given by World vocoder [24] for 100 epochs, and the ASR model is trained at phoneme level for 80 epochs with a character error rate (CER) of 8.53%. AUTO-VC is trained with one-hot embedding for 1M steps.

ref to https://github.com/thsno02/StarGANv2-VC/blob/ed20538971a03d699351a349a3631767333baeb7/Configs/config.yml#L34

can't find $\lambda_{cls}$ in the code?

**Comment by thsno02** [comment]
# Number of Speakers

h tps://github.com/yl4579/StarGANv2-VC/iss ues/17
>I would say 10-20 speakers would be the best setting

In the original paper, the authors used 10 speakers and achieved an amazing performance. I may use 10 too.

# Batch Size

h tps://github.com/yl4579/StarGANv2-VC/iss ues/36
> As for the batch size, a larger batch size might cause the discriminator to dominate because they see more samples and could result in worse generator performance, that's why in GAN training the batch size is usually set to be very small (generally less than 10). You can try to change the lambda for the adversarial loss though to make the training more balanced.

Use 5 as the author did.

# Other Hyper-parameters

>The hyperparameters barely have any effects in my experience, those were chosen arbitrarily and could possibly be anything reasonable (e.g., as long as you don't start training adversarial classifiers or data augmentation right away).

No need to tune hyper-parameters for now.
